fileutil/mmap_windows.go

```diff
@@ -35,7 +35,7 @@ func mmap(f *os.File, sz int) ([]byte, error) {
 		return nil, os.NewSyscallError("CloseHandle", err)
 	}
 
-	return (*[1 << 30]byte)(unsafe.Pointer(addr))[:sz], nil
```

Comment:
`sz` is just `size`? Can we name it like that? (maybe just for master)

Reply:
Also I have stupid question. I have never used `mmap` before, but... why don't we cast to just `[sz]byte` here?

Reply:
I was thinking the same thing and tried it but casting to an array requires a constant for the array size as this is evaluated at compile time

-	return (*[sz]byte)(unsafe.Pointer(addr))[:sz], nil	=  non-constant array bound